api/client.go

```diff
@@ -240,16 +234,6 @@ func (c *Config) ConfigureTLS() error {
 // This also removes any trailing "/v1"; we'll use that in our commands so we
 // don't require it from users.
 func (c *Config) setAddr(addr string) error {
-	if strings.HasPrefix(addr, "unix://") {
```

Comment:
This seems to ignore the comment on the field you removed

Reply:
Oh, the comment says that the field takes a filename approach instead of a URL approach since parsing file names as a URL may error out on some platforms -- and then it removes the code that keeps it from being parsed as a URL. :-)

Reply:
Confirmed this will be tracked separately, resolving